app/gui/docs/product/visualizations.md

```diff
@@ -297,9 +283,8 @@ visualization you may use the `setPreprocessor` method). This code defines an
 Enso function, which will be run by the compiler on data the visualization is
 attached to. Only the results of this code will be sent to the GUI. In the case
 of the JSON input format, the result of the call should be a valid JSON string.
-The code will be evaluated in the context of the `Main` module in the project
-where visualization is defined - you may use any symbol defined or imported in
-that module.
+The code will be evaluated in the context of the module where the preprocessor
+method is defined - you may use any symbol defined or imported in that module.
```

Comment:
How can one pass arguments to the method? #3655 needs to be able to invoke the Enso engine part of the visualization with different arguments.

Reply:
I'd expect that arguments are part of the expression, as the arguments change we update the expression.

Reply:
Ah, it was removed. Then some mechanism for arguments is necessary,